orca-core/orca-core.gradle

```diff
@@ -30,4 +30,5 @@ dependencies {
   compile spinnaker.dependency('kork')
   compile ("commons-collections:commons-collections:3.2.1")
   testCompile project(":orca-test")
+  compile "com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.6.3"
```

Comment:
I added a proper alias for this to spinnaker/spinnaker-dependencies but Orca is currently behind the latest release of that.